config/desktop-development.json

```diff
@@ -73,7 +73,6 @@
 		"manage/edit-user": true,
 		"manage/export/guided-transfer": true,
 		"manage/import/medium": true,
-		"manage/option_sync_non_public_post_stati": true,
```

Comment:
Was this a typo or was it written by someone fluent in Latin? :smile:

Reply:
Haha! Well, funny enough, post stati [is a thing in WordPress](https://codex.wordpress.org/Function_Reference/get_post_stati).